tests/common/fixtures/services.py

```diff
@@ -23,8 +24,10 @@
 
 @pytest.fixture
 def mock_service(pyramid_config):
-    def mock_service(service_class, name):
-        service = create_autospec(service_class, instance=True, spec_set=True)
+    def mock_service(service_class, name, spec_set=True, **kwargs):
+        service = create_autospec(
+            service_class, instance=True, spec_set=spec_set, **kwargs
+        )
```

Comment:
Changed this to allow arbitrary kwargs to be passed in and to allow spec_set to be overridden, see below for why